src/api/users.ts

```diff
@@ -32,6 +32,18 @@ usersRouter.get("/me", requireUser, async (req, res, next): Promise<void> => {
     }
 })
 
+// GET /api/users
+usersRouter.get("/", async (req, res, next): Promise<void> => {
+    // if (req.user) {
+        try {
+            const users = await prisma.user.findMany();
+            res.send({users});
```

Comment:
Security breach! This is sending back all of the users' passwords. Remember to delete passwords before sending!!

Reply:
Thanks for the catch!